src/documentation/documentation.html

```diff
@@ -2,10 +2,12 @@
   <div class="page animated-page au-animate documentsContainer">
     <div class="sidebar">
       <template repeat.for="doc of router.navigation">
-        <a href.to-view="doc.href"
-          class="document ${doc.isActive ? 'active' : ''}">
-          ${doc.title}
-        </a>
+        <div>
```

Comment:
better than adding an unnecessary div is to add `display: block` to the `a` (in the scss)